cmd/soroban-cli/src/commands/tx/new/create_passive_sell_offer.rs

```diff
@@ -0,0 +1,75 @@
+use clap::{command, Parser};
+
+use crate::{commands::tx, tx::builder, xdr};
+
+#[derive(Parser, Debug, Clone)]
+#[group(skip)]
+pub struct Cmd {
+    #[command(flatten)]
+    pub tx: tx::Args,
+
+    #[clap(flatten)]
+    pub op: Args,
+}
+
+#[derive(Debug, clap::Args, Clone)]
+pub struct Args {
+    /// Asset to sell
+    #[arg(long)]
+    pub selling: builder::Asset,
```

Comment:
Nit pick: not sure how useful it would be, but I wonder if we could add a bit more description about how to input the assets, like perhaps giving possible values: "i.e. `native`, `BTC:GAAA...COINBASE`, etc".

Definitely doesn't have to be part of this PR because I think that we'd need to make this change to several commands, but just a thought.

Reply:
good idea! I'll update asset descriptions everywhere once all these commands are merged.

Reply:
Also aliases can be used for hte issuer part of the name, so good to demo that too, it's easy to forget that's possible.